spec/features/user_adds_api_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+# [ ] I want to add a new API So others can see and review it
+
+feature "user adds a new api" do
+  let!(:api1) {
```

Comment:
Personally I disagree with this and think it's fine. You can safely ignore these warnings.